numpy/core/src/umath/simd.inc.src

```diff
@@ -3695,6 +3728,81 @@ sse2_@kind@_BOOL(@type@ * op, @type@ * ip, const npy_intp n)
 
 #undef VECTOR_SIZE_BYTES
 
-#endif /* NPY_HAVE_SSE2_INTRINSICS */
+#else  /* NPY_HAVE_SSE2_INTRINSICS */
+/**begin repeat
+ *  #type = npy_float, npy_double#
+ *  #TYPE = FLOAT, DOUBLE#
+ *  #sfx = f32, f64#
+ *  #CHK =    , _F64#
+ */
+#if NPY_SIMD@CHK@
+
+/**begin repeat1
+* Arithmetic
+* # kind = add, subtract, multiply, divide#
+* # OP = +, -, *, /#
+* # VOP = add, sub, mul, div#
+*/
 
+static void
+simd_binary_@kind@_@TYPE@(@type@ * op, @type@ * ip1, @type@ * ip2, npy_intp n)
+{
+    LOOP_BLOCK_ALIGN_VAR(op, @type@, NPY_SIMD_WIDTH)
+        op[i] = ip1[i] @OP@ ip2[i];
```

Comment:
Can you add braces to match the `LOOP_BLOCKED_END`?

Reply:
Braces added as required.